### PR TITLE
Use only English in public interface

### PR DESCRIPTION
**Pull request by gbirke**
Renamed all the properties and one entity.
Updated version to 2.0 because this is a BC-breaking change.

Please review, test with the other applications, create pull requests for them and then merge.
Or create a 2.0 version that can be used in parallel in the new FundraisingFrontend

### CODE REVIEW

**Comment by KaiNissen** [comment]
I'm +1 on this. All that serialized base64 encoded *stuff* in `Donation::data` will still have German array keys. Of course, that's not opposing this change, but we might want to think about that as well.

**Comment by wmde-manicki** [comment]
Major version change every day!

@gbirke I might be confusing things in the morning but as this is indeed a BC-breaking change, and you change the version number plus composer's branch alias to 2.0 (2.0.x-dev), all things using those entity classes would not even notice this change unless told explicitly to use version 2+ (all three have requirement for version `~1.0` which if I understand it right wouldn't accept `2.0.x-dev`). So this could be merged and released if wanted, and all usages could be then changed when it is wanted/needed (for instance I believe the old fundraising app would not be the first to change). Or is this not that beautiful?

@KaiNissen this is important thing to have in mind. One way to go would be to have a set of public methods accessing some stuff from `Donation::data` instead of always playing with getting the encoded data and accessing some of its keys in "third-party" code. I am not sure whether those methods should be a part of this entity class, though.
E.g. having something like `Foo::getFirstName()` instead of always calling `Donation::getData()`, then unserializing the result, and finally accessing `$foo['vorname']`. This way there would not be a need to touch `data`'s key names (I guess to would not be wise).
And that way I could finally type `getAddressType()` instead of fixing typos in `adresstyp` several times each time I try to access that field :)
Actually there seems to be already something like that, see `Donation::getUExpiry()`.

But to the main point: I'd be +1 for this change too. I cannot tell much about the English equivalents of German names, that's up to you guys. I can't even recall where this `bescheinigung` is used and what for. With `eintrag` I recall it is the name people would like to have their comments published under, right?

One remark: shouldn't all newly named classes other than `Donation` also get `@since 2.0` tags?

File: README.md [hunk 1]
```diff
@@ -44,6 +44,12 @@ run style checks use `composer cs`.
 
 ## Release notes
 
+### Version 2.0 (2016-01-13)
+
+* Rename entity `Spenden` to `Donation`. Database table name stays the same.
+* Use only english property names in public interface of `Request` and `Donation` class.
```
Comment (wmde-manicki) [inline README.md]: English

**Comment by wmde-manicki** [comment]
This also fixes https://github.com/wmde/FundraisingStore/issues/7

**Comment by gbirke** [comment]
Thanks for all the kind remarks! I'll fix the two remaining issues, create a branch for the 1.0.x-dev version (so changes and new 1.x releases can be made). That should keep the old applications compatible.

Regarding the `data` field: Do we have some documentation what is stored there with which keys and what keys are actually used at the moment?

**Comment by gbirke** [comment]
Closing the PR since we decided to create a dedicated 2.0 development branch

**Comment by KaiNissen** [comment]
new PR is #35

**Pull request closed without merging**